frontend/src/main.jsx

```diff
@@ -13,6 +15,8 @@ root.render(
       <Routes>
         <Route path="/" element={<App />} />
         <Route path="/offers/:id" element={<OfferDetails />} />
+        <Route path="/connexion" element={<ConnexionPage />} />
+        <Route path="/createentreprise" element={<CreateAccount />} />
```

Comment:
use more restful conventions: `/companies/create` for example

Reply:
route name was changed to more restful conventions